awx/main/models/projects.py

```diff
@@ -199,7 +199,7 @@ def playbooks(self):
         results = []
         project_path = self.get_project_path()
         if project_path:
-            for dirpath, dirnames, filenames in os.walk(smart_str(project_path)):
+            for dirpath, dirnames, filenames in os.walk(smart_str(project_path),followlinks=settings.AWX_SHOW_PLAYBOOK_LINKS):
```

Comment:
This is a bit nit-picky style-wise, but it would be nice to have a space after the comma `, followlinks=`